kratos/KratosMultiphysics.json

```diff
@@ -1,11 +1,10 @@
- 
 {
 	"wheel_name": "KratosMultiphysics",
 	"included_modules": [],
-	"included_binaries": ["Kratos.cpython-${PYTHON}-x86_64-linux-gnu.so"],
+	"included_binaries": ["Kratos.*", "KratosCore.*", "KratosCore.*", "zlib.*", "libz.*"],
```

Comment:
is that all?
What about e.g. tinyexpr which was added some time ago?

Reply:
As far as I can tell you it does not generate a separated .so / .dll, so no need to add it here.

But since you mention this, I think mmg / hdf5 app should ship with the third party *.so. I will have those fixed thx!

Reply:
for me it generates `libtinyexpr.a` (on linux)

Ah maybe not on Win?

Reply:
a but that is the static lib, it's a byproduct of the compilation process, no need to include it.